PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskTrackSkim.cxx

```diff
@@ -411,6 +419,7 @@ Bool_t AliAnalysisTaskTrackSkim::FillHistograms()
     }
 
     for (auto particle : particlesGen->accepted()) {
+      fSecondTrackCollectionKinematics["pt"].push_back(particle->Pt()); 
```

Comment:
Seems to be here twice - this is a bug? Did you mean to do the same as for the first track collection? I would think you would want the charge at both det and particle level

Reply:
Indeed a bug.. my bad! The idea for particle level is to extract the charge from PDG code

Reply:
Ah, yeah, of course